play-services-basement/src/main/java/org/microg/gms/common/GmsService.java

```diff
@@ -19,7 +19,7 @@
 public enum GmsService {
     UNKNOWN(-2),
     ANY(-1),
-    GAMES(1, "com.google.android.gms.games.service.START", "com.google.android.gms.games.internal.connect.service.START"),
+    GAMES(1, "com.google.android.gms.games.internal.connect.service.START", "com.google.android.gms.games.service.START"),
```

Comment:
This would likely break the non-connect service. I guess the issue might be in the service provider. Will investigate

Reply:
Yes, if the GmsService class specifies that a serviceid contains multiple actions, ServiceProvider will take the first action by default, but in fact it is another action that needs to be jumped. For example, in the game service, GamesConnectService has a higher priority than GamesService, but in In the call method of ServiceProvider, GamesService will be used

Reply:
I removed that part since it's no longer needed with 64307c1c194d21f5794a5d8789ecdd280b301e7d